File: Java/src/e31/e_36.java
package e31;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class e_36 {
    public static void main(String[] args) {
        // Crear la lista original
        List<Double> L = Arrays.asList(-4.2, 2.3, 8.0, -2.0, 6.8);

        // Crear la nueva lista L1 mediante un bucle
        List<Double> L1 = new ArrayList<>();
        for (double x : L) {
            if (x > 0) {
                L1.add(x);
            }
        }

        // Imprimir la lista L1
        System.out.println("Lista L1: " + L1);
    }
}

// En este ejemplo, Arrays.asList se utiliza para crear una lista original L y
// un bucle for-each se utiliza para recorrer la lista original y agregar cada
// número positivo o mayor que 0 a la nueva lista L1. La nueva lista L1 se
// imprime al final del programa. Ten en cuenta que este ejemplo asume que los
// números son de tipo double, pero puedes ajustarlo según tus necesidades
// específicas.

File: Python/31-40/e_36.py
# Escribe en python un programa para crear la lista L = [-4.2, 2.3, 8, -2, 6.8]. 
# A continuación crea una nueva lista L1 usando comprensión de lista con los números de L que son positivos o mayores que 0. 
# Imprime la lista L1.


# Crear la lista original
L = [-4.2, 2.3, 8, -2, 6.8]

# Crear la nueva lista L1 mediante comprensión de lista
L1 = [x for x in L if x > 0]

# Imprimir la lista L1
print("Lista L1:", L1)


# Puedes crear la nueva lista L1 utilizando una comprensión de lista que incluya solo los números positivos o mayores que 0 de la lista original L.
# En este ejemplo, la comprensión de lista [x for x in L if x > 0] selecciona cada elemento x de la lista L solo si es positivo o mayor que 0,
# creando así la nueva lista L1. La lista L1 se imprime al final del programa.





